subprojects/internal-performance-testing/src/main/groovy/org/gradle/performance/AbstractCrossVersionGradleProfilerPerformanceTest.groovy

```diff
@@ -43,7 +45,7 @@ import spock.lang.Specification
 @CleanupTestDirectory
 class AbstractCrossVersionGradleProfilerPerformanceTest extends Specification {
 
-    private static def resultStore = new CrossVersionResultsStore()
+    private static final RESULTS_STORE = createResultsStoreWhenDatabaseAvailable { new CrossVersionResultsStore() }
```

Comment:
Why no type declaration here?

Reply:
Because IDEA can infer the type. For some reason, it can't infer the type in `AbstractGradleVsMavenPerformanceTest`, that is why I only added it there.